doc/gazebo_simulation/gazebo_simulation.rst

```diff
@@ -0,0 +1,351 @@
+Gazebo Simulation Integration
+=============================
+
+`Gazebo <http://gazebosim.org/>`_ is the most popular robotics simulator in the ROS ecosystem, so is naturally a good fit to integrate with MoveIt.
+
+The `MoveIt Setup Assistant <../setup_assistant/setup_assistant_tutorial.html>`_ helps setup your robot to work with Gazebo, but there are still additional steps required to successfully run MoveIt in Gazebo.
+
+----------------------------
+After MoveIt Setup Assistant
+----------------------------
+This tutorial assumes that the robot is set up with `MoveIt Setup Assistant <../setup_assistant/setup_assistant_tutorial.html>`_,
+so it is crucial to follow that document first. To the best of our knowledge, official FRANKA EMIKA Panda repository doesn't particularly consider Gazebo simulation
+such that the necessary components to properly simulate the robot in Gazebo are missing with respect to :code:`urdf` and :code:`xacro` files. This is a rare incident, since most other robots
+have those components out of the box. *If you have a custom robot, which already works well in Gazebo, you can skip the steps until Step-6.* Fortunately, there is already a good solution offered in `the blog post <https://erdalpekel.de/?p=55>`_ to this problem. For the sake of completion though,
+the procedure outlined in there will be repeated (with improvements) in here as well for preparing the robot for Gazebo simulation.
+**Note that these steps assume that you have cloned franka_ros repository from the source**:
```

Comment:
It would be helpful to include a link to the repository.

Reply:
It assumes that the user had followed the Setup Assistant tutorial, which has that link.

Reply:
```suggestion
**Note that these steps assume that you have cloned the `franka_ros repository <https://github.com/frankaemika/franka_ros>` from the source**:
```

Reply:
It does have the link, but it also recommends installing the package via `apt-get`, so I didn't actually need the link until I got to this step, which is why I suggest adding the link.